controllers/phjob_scheduler.go

```diff
@@ -291,7 +299,7 @@ func (r *PHJobScheduler) scheduleByStrictOrder(phJobsRef *[]*primehubv1alpha1.Ph
 
 		_, foundUser := usersRemainingQuota[phJob.Spec.UserName]
 		if !foundUser {
-			userRemainingQuota, err := r.getUserRemainingQuota(phJob)
+			userRemainingQuota, err := r.getUserRemainingQuotaInGroup(phJob)
 			if err != nil {
 				return err
```

Comment:
should it be ?

```go
return nil, err 
```

Reply:
Here we just return err based on scheduleByStrictOrder function.